Tutorial-Importing-the-Movie-Database-from-Neo4j.md

```diff
@@ -0,0 +1,255 @@
+Type 'help' to display all the supported commands.
+Installing extensions for GREMLIN language v.2.6.0
+
+orientdb> CREATE DATABASE PLOCAL:D:/orientdb/orientdb-enterprise-2.2.8/databases/movie
+
+Creating database [PLOCAL:D:/orientdb/orientdb-enterprise-2.2.8/databases/movie] using the storage type [PLOCAL]...
+Database created successfully.
+
+Current database is: PLOCAL:D:/orientdb/orientdb-enterprise-2.2.8/databases/movie
+orientdb {db=movie}> IMPORT DATABASE D:/movie.graphml
+
+Importing GRAPHML database from D:/movie.graphml with options ()...
+Done: imported 171 vertices and 253 edges
+orientdb {db=movie}>
+```
+
+As you can see from the output above, as a result of the import 171 vertices and 253 edges have been created in OrientDB. This is exactly the same number of nodes and relationships exported from Neo4j.
+
+For more tips and tricks related to the import process, please refer to [this](Import-from-Neo4j-into-OrientDB.md#import-tips-and-tricks) section.
+
+
+## Query Comparison
+
+Once the *movie* database has been imported into OrientDB, you may use several ways to access its data.
+
+The `MATCH` [syntax](SQL-Match.md) and the tool [Studio](Studio-Home-page.md) can be used, for instance, in a similar way to the Neo4j's Cypher and Browser.
+
+The following tables include a comparison of the Neo4j's Cypher and OrientDB's `MATCH` syntax for some queries that you can execute against the *movie* database:
+
+<table>
+<tr>
+	<th>Objective</th>
+    <th>Neo4j's Cypher</th>
+    <th>OrientDB's MATCH</th>
+</tr>
+<tr>
+<td>Find the actor named "Tom Hanks"</td>
+<td>
+<pre>
+MATCH (tom:Person {name: "Tom Hanks"}) 
+RETURN tom
+</pre>
+</td>
+<td>
+<pre>
+MATCH {class: Person, as: tom, where: (name = 'Tom Hanks')} 
+RETURN $pathElements
+</pre>
+</td>
+</tr>
+</table>
+<table>
+<tr>
+	<th>Objective</th>
+    <th>Neo4j's Cypher</th>
+    <th>OrientDB's MATCH</th>
+</tr>
+<tr>
+<td>Find the movie with title "Cloud Atlas"</td>
+<td>
+<pre>
+MATCH (cloudAtlas:Movie {title: "Cloud Atlas"}) 
+RETURN cloudAtlas
+</pre>
+</td>
+<td>
+<pre>
+MATCH {class: Movie, as: cloudAtlas, where: (title = 'Cloud Atlas')} 
+RETURN $pathElements
+</pre>
+</td>
+</tr>
+</table>
+<table>
+<tr>
+	<th>Objective</th>
+    <th>Neo4j's Cypher</th>
+    <th>OrientDB's MATCH</th>
+</tr>
+<tr>
+<td>Find 10 people</td>
+<td>
+<pre>
+MATCH (people:Person) 
+RETURN people.name 
+LIMIT 10
+</pre>
+</td>
+<td>
+<pre>
+MATCH {class: Person, as: people} 
+RETURN $pathElements
+LIMIT 10
+</pre>
+</td>
+</tr>
+</table>
+<table>
+<tr>
+	<th>Objective</th>
+    <th>Neo4j's Cypher</th>
+    <th>OrientDB's MATCH</th>
+</tr>
+<tr>
+<td>Find the movies released in the 1990s</td>
+<td>
+<pre>
+MATCH (nineties:Movie) 
+WHERE nineties.released > 1990 AND nineties.released < 2000 
+RETURN nineties.title
+</pre>
+</td>
+<td>
+<pre>
+MATCH {class: Movie, as: nineties, WHERE: (released > 1990 AND released < 2000 )} 
+RETURN $pathElements
```

Comment:
In this case you can use 

```
RETURN nineties.title
```

in OrientDB as well, the result will be much more similar to the original one